rocketmq-flink/src/main/java/org/apache/rocketmq/flink/RocketMQSink.java

```diff
@@ -59,7 +60,9 @@
 
     private Properties props;
     private TopicSelector<IN> topicSelector;
-    private KeyValueSerializationSchema<IN> serializationSchema;
+    private KeyValueSerializationSchema<IN> keyValueSerializationSchema;
+    //flink sql connector serialization schema
+    private RmqSerializationSchema<IN> rmqSerializationSchema;
```

Comment:
RmqSerializationSchema is a confusing name. You Could change it to a self-explanation name such as SQLSerializationSchema class or field.